tareas/clase-5/tarea-clase-5-1.js

```diff
@@ -22,6 +22,17 @@ $botonCalcular.onclick = function()
 //cuando el usuario haga click en el botón "calcular", mostrar el salario mensual
 // en una caja de texto deshabilitada. --> <input type="text" disabled id="salario-mensual"/>
 
+function calcularSalarioMensual(salarioAnual){
+  return salarioAnual / 12;
+}
+
+document.querySelector("#calcular-salario-mensual").onclick = function $calcularSalarioMensual(){
```

Comment:
`$calcularSalarioMensual` da a entender que estás guardando un elemento, no debería ser el nombre de la función, que tranquilamente puede ser anónima.

```suggestion
document.querySelector("#calcular-salario-mensual").onclick = function (){
```